Makefile

```diff
@@ -0,0 +1,59 @@
+
+PYTEST_IGNORES := -W ignore::pytest.PytestCollectionWarning
+
+########################################
+##@ General
+default: help
+
+help: ## Print this message
+	@echo "====================="
+	@echo "🛠  Available Commands"
+	@echo "====================="
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-25s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+prereq: ## Install the development prerequisites
+	pip install -r ./custom_model_runner/requirements.txt -r requirements_test_unit.txt -r requirements_test.txt -r requirements_lint.txt
```

Comment:
the `pip install -r ./custom_model_runner/requirements.txt` will be installed in the next line by `pip install -e custom_model_runner/`. Looks like a duplicate installation?